weblab_cg/__init__.py

```diff
@@ -29,7 +29,21 @@
     load_template,
 )
 
+from ._generator_chaste_cg import (   # noqa
+    NormalChasteModel,
+    OptChasteModel,
+    Analytic_jChasteModel,
+    Numerical_jChasteModel,
+    BEOptChasteModel,
+)
+
 from ._printer import (     # noqa
     WebLabPrinter,
+    _math_functions,
```

Comment:
Shouldn't usually import hidden things...

Reply:
Fair point I made it a class level variable in the printer so I have access to it